app/forms/course_details_form.rb

```diff
@@ -91,7 +91,7 @@ def other_age_range?
 
   def update_trainee_attributes
     trainee.assign_attributes({
-      course_code: nil,
+      course_code: course_code,
```

Comment:
We still want to set this to nil if the "Another course not listed" option has been selected then the course code is redundant in that scenario